README.md

```diff
@@ -1,6 +1,6 @@
 # torchtrtz
 
-This project is mainly used t generate tensorrt weights for [makaveli10/cpptensorrtz](https://github.com/makaveli10/cpptensorrtz)
+This project is mainly used to generate tensorrt weights for [makaveli10/cpptensorrtz](https://github.com/makaveli10/cpptensorrtz)
```

Comment:
shouldn't this be a different commit?

Reply:
It's a different commit right? Update: readme ?
There are 2 commits. Am I missing something?

Reply:
Oh sorry, I missed that.